evaluator/build.gradle.kts

```diff
@@ -33,6 +33,7 @@ dependencies {
 
     implementation("org.jetbrains.kotlin:kotlin-scripting-common")
     implementation("org.jetbrains.kotlin:kotlin-scripting-jvm-host")
+    implementation(libs.kotlinxSerialization)
```

Comment:
This commit should be squashed into the previous commit, why not use the easier approach right away?

Reply:
Out of sentimentality, I guess 😆 I started this PR when there was no JSON but only the CSV version, and I did put some effort into the correct implementation that I wanted to keep, just in case someone ever needs to process and old CSV version again.

But I agree it's a bit inconvenient to review this way, also see my [initial note](https://github.com/oss-review-toolkit/ort/pull/4977#issue-1108001840) again. Do you prefer me to squash the JSON implementation into the CSV one?

Reply:
> Do you prefer me to squash the JSON implementation into the CSV one?

I've done that.